ocaml/xapi/vgpuops.ml

```diff
@@ -146,32 +145,43 @@ let create_virtual_vgpu ~__context vm vgpu =
 				~self:vgpu.vgpu_ref ~value:pgpu;
 	)
 
-let add_vgpus_to_vm ~__context vm vgpus =
+let add_vgpus_to_vm ~__context vm vgpus host =
 	(* Only support a maximum of one virtual GPU per VM for now. *)
 	match vgpus with
 	| [] -> ()
-	| vgpu :: _ -> create_virtual_vgpu ~__context vm vgpu
+	| vgpu :: _ -> create_virtual_vgpu ~__context vm vgpu host
 
 let vgpu_manual_setup_of_vm vm_r =
 	List.mem_assoc Xapi_globs.vgpu_manual_setup_key vm_r.API.vM_platform &&
 	(List.assoc Xapi_globs.vgpu_manual_setup_key vm_r.API.vM_platform = "true")
 
-let create_vgpus ~__context (vm, vm_r) hvm =
+let get_vgpus ~__context (vm, vm_r) hvm passthrough =
 	let vgpus = vgpus_of_vm ~__context vm_r in
-	if vgpus <> [] then begin
```

Comment:
Did you mean to reindent this section? It looks like the tabs have just been swapped for spaces with no code change.